nin/frontend/app/scripts/services/demo.js

```diff
@@ -25,6 +26,16 @@ function demo(commands, $rootScope, $window) {
     stats[i].dom.style.float = 'right';
     stats[i].dom.style.display = 'block';
   }
+
+  commands.on('toggleStats', () => {
+    showStats = !showStats;
+    for(let i = 0; i < 3; i++) {
+      stats[i].dom.style.display = showStats ? 'block' : 'none';
```

Comment:
If you use the current display style you can get away with no local style foro flipping the stats.